orbit/cmd/orbit/orbit.go

```diff
@@ -938,7 +937,7 @@ func main() {
 			orbitClient.RegisterConfigReceiver(update.ApplyWindowsMDMEnrollmentFetcherMiddleware(windowsMDMEnrollmentCommandFrequency, orbitHostInfo.HardwareUUID, orbitClient))
 			orbitClient.RegisterConfigReceiver(update.ApplyWindowsMDMBitlockerFetcherMiddleware(windowsMDMBitlockerCommandFrequency, orbitClient))
 		case "linux":
-			orbitClient.RegisterConfigReceiver(luks.New(orbitClient, zenity.New()))
```

Comment:
moving this to the config reciever as technically you can have zenity installed on a kubuntu system (and that's the better UX)